scripts/bert/embedding.py

```diff
@@ -56,24 +66,41 @@ def __init__(self, ctx=mx.cpu(), model='bert_12_768_12',
         ----------
         ctx : Context.
             running BertEmbedding on which gpu device id.
+        dtype: str
+            data type to use for the model.
         model : str, default bert_12_768_12.
             pre-trained BERT model
         dataset_name : str, default book_corpus_wiki_en_uncased.
             pre-trained model dataset
+        params_path: str, default None
+            path to a parameters file to load instead of the pretrained model.
         max_seq_length : int, default 25
             max length of each sequence
         batch_size : int, default 256
             batch size
         """
         self.ctx = ctx
+        self.dtype = dtype
         self.max_seq_length = max_seq_length
         self.batch_size = batch_size
+        self.dataset_name = dataset_name
         self.bert, self.vocab = gluonnlp.model.get_model(model,
-                                                         dataset_name=dataset_name,
+                                                         dataset_name=self.dataset_name,
                                                          pretrained=True, ctx=self.ctx,
```

Comment:
Shall we set `pretrained=False` if `params_path` is present? This avoids downloading the pretrained parameters

Reply:
Didn't realise that's what it's doing, will fix.